app/src/main/java/org/oppia/app/topic/review/ReviewSkillSelectionAdapter.kt

```diff
@@ -0,0 +1,67 @@
+package org.oppia.app.topic.review
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.library.baseAdapters.BR
+import androidx.recyclerview.widget.RecyclerView
+import org.oppia.app.databinding.TopicReviewSummaryViewBinding
+import org.oppia.app.model.SkillSummary
+import com.bumptech.glide.request.RequestOptions
+import com.bumptech.glide.Glide
+import androidx.databinding.BindingAdapter
+import android.widget.ImageView
+import org.oppia.app.R
+
+// TODO(#216): Make use of generic data-binding-enabled RecyclerView adapter.
+/** Adapter to bind skills to [RecyclerView] inside [TopicReviewFragment]. */
+class ReviewSkillSelectionAdapter(private val reviewSkillSelector: ReviewSkillSelector) :
+  RecyclerView.Adapter<ReviewSkillSelectionAdapter.SkillViewHolder>() {
+
+  companion object {
+    @JvmStatic
+    @BindingAdapter("android:src")
+    fun loadSkillThumbnail(view: ImageView, imageUrl: String) {
```

Comment:
I suggest either moving this to a general purpose Kotlin file & rename it to be generic like we do for other binding adapters in #195 or change the adapter string to be skill-specific (e.g. ``"skill_thumbnail"``) to avoid this being unintentionally used for non-skill thumbnail cases.

Reply:
Removed this `companion object` and introduced `ImageBindingAdapter` from #195